scopes/mdx/ui/mdx-layout/mdx-layout.tsx

```diff
@@ -84,7 +85,11 @@ export type MDXLayoutProps = {
  */
 export function MDXLayout({ children, mdxComponents }: MDXLayoutProps) {
   const components = Object.assign(defaultMdxComponents, mdxComponents);
-  return <MDXProvider components={components}>{children}</MDXProvider>;
+  return (
+    <MDXProvider components={components}>
+      <div className={styles.mdxContent}>{children}</div>
```

Comment:
did this to prevent overflowing youtube iframe (and future stuff) in mdx cards and layouts